File: programminglanguages/Assignment1/task1.rb
#Author: Thomas Curtis
#Program 1: Takes user from input and writes data to file, then the program reads the file and outputs the number of character in the file 

#Asking the user to enter three lines of words
puts "Enter three lines of words"
line0 = gets
line1 = gets
line2 = gets

#Creating a new file and putting the contents on line0..line2 in it
aFile = File.new("myFile.txt", "w")
aFile.write(line0,line1,line2)
aFile.close

#Reading the file("myFile.txt") and count the total number of characters
aFile = File.open("myFile.txt", "r")
lines = File.readlines("myFile.txt")
numLines = lines.size
char = lines.join
totalChar = char.length
puts "#{totalChar} characters"
aFile.close

File: programminglanguages/Assignment3/task1.scala
//Author: Thomas Curtis
//Description: The program ask a user for a file name and then writes to the file. 

import scala.io.StdIn.readLine
import java.io.File
import java.io.PrintWriter
import scala.io.Source

object task1 { 
	
	def main(args: Array[String]):Unit  = { 

		val file = new PrintWriter(new File("file.txt")) //gets file to write
		var x = 0
		while(x < 3) {
			var line = scala.io.StdIn.readLine() //input from user
			file.write(line + "\n")
			x = (x + 1)
		}
		file.close()
    }
}   

